components/common/RichText.tsx

```diff
@@ -52,6 +52,22 @@ type RichTextImageProps = {
   };
 };
 
+const RichTextStyles = styled.div`
```

Comment:
Because these styled components are still components, I wouldn't name it as just --Styles, even thought styling is here the only reason for the component's existence. I would maybe call these StyledRichText or RichTextContainer or something like that. (I think we have used at least Styled* elsewhere.

Reply:
Changed. Thank you for a suggestion.